src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -285,11 +279,9 @@ Copyright (c) .NET Foundation. All rights reserved.
   <!-- Back-compat for PrecompilationTool -->
   <PropertyGroup>
     <!--
-      For 2.x desktop targeting projects using MvcPrecompilation, ref assemblies are required to compile the PrecompiledViews.dll,
-      but are removed by the tool once done. Set PreserveCompilationReferences = true, ignoring any value configured in the project,
-      if the project is using the precompilation tool.
+      Always enable PreserveCompilationReferences for non 3.0 projects. This preserves parity with the 2.x SDK's behavior.
     -->
-    <PreserveCompilationReferences Condition="'$(ResolvedRazorCompileToolset)'=='PrecompilationTool'">true</PreserveCompilationReferences>
+    <PreserveCompilationReferences Condition="'$(_TargetingNETCoreApp30OrLater)' != 'true'">true</PreserveCompilationReferences>
```

Comment:
Will this now impact Blazor client side scenarios?

Reply:
Yup. They'll have refs directories in their outputs.

Reply:
🆙 📅 this after talking to the powers to be. We decided we'd limit this change to .NET Fx and .NET Core < 2.x projects. While it's a breaking change, having ref assemblies in ns2.0 isn't very useful in the ordinary cases.

Reply:
(I don't know how to read those symbols.)